README.md

```diff
@@ -450,32 +435,41 @@ When the code is ready, submit a Pull Request.
 
 ### Running Tests
 
-`trino-python-client` uses [pytest](https://pytest.org/) for its tests. To run
+`trino-python-client` uses [tox](https://tox.wiki/en/latest/)—a tool for standardizing testing in Python—which leverages the [pytest](https://pytest.org/) testing framework. To run
 only unit tests, type:
 
 ```
-$ pytest tests/unit
+$ poetry run tox -e <environment> -- tests/unit
```

Comment:
@hashhar local development likely wants to run multiple environments, i.e., `py39`, `pre-commit`, etc. Personally having used `tox` for a number of years I have no issue with explicitly invoking the Python specific environment when running the unit/integration tests.

Reply:
We should reduce friction for new contributors. Generally people shouldn't need to care what env they are testing against unless the issue is env specific (then they can provide their own choice of env).

Reply:
I'm not really sure explicitly specifying an environment is friction. It's worth noting that local CI likely requires multiple environments, i.e., a common workflow could be to first run `tox -e pre-commit` to ensure linting, typing, etc. is correct before running the test via `tox -e py39` say.

One could argue having a these environments—a subset of which can be run via the `-e` argument—actually reduces friction as it obfuscates the actual commands from the user, i.e., rather than the user needing to know they need to run the `pytest`, `pre-commit`, etc. commands (along with the relevant flags/arguments) they simply invoke the respective environment through the `tox` CLI which standardizes testing workflows.

Reply:
Perhaps adding some examples like `poetry run tox -e py39 -- tests/unit` could be a reasonable consensus? Contributors usually follow README first, or we can have a sample local testing workflow described like run pre-commit then running tests etc.